xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -825,7 +827,7 @@ unsigned int CAESinkAUDIOTRACK::AddPackets(uint8_t **data, unsigned int frames,
     const double max_stuck_delay_ms = m_audiotrackbuffer_sec_orig * 2000.0;
     const double stime_ms = 1000.0 * frames / m_format.m_sampleRate;
 
-    if (m_stuckCounter * stime_ms > max_stuck_delay_ms)
+    if (!m_ignoreBrokenAudioDelay && (m_stuckCounter * stime_ms > max_stuck_delay_ms))
```

Comment:
I find this a bit unintuitive as default advancedsetting is enabled and here negated and in `AESinkAUDIOTRACK.h` variable is initialized to false.

Maybe reword to "m_trackAudioDelay" or similar and default = false on all places (if the intention is not track by default).

Reply:
Infact I did not think too much about it ... I hate such workarounds ... ;-) but your suggestion is good.